code/game/objects/items/storage/boxes.dm

```diff
@@ -141,6 +141,8 @@
 	new /obj/item/clothing/mask/gas/explorer(src)
 	new /obj/item/tank/internals/emergency_oxygen(src)
 	new /obj/item/crowbar/red(src)
+	var/obj/item/gps/mining/snowflake = new /obj/item/gps/mining(src)
+	snowflake.gpstag = src.loc.loc.name
```

Comment:
wtf is this?

also remove it from the mining base if you're giving them one

Reply:
it will be removed from the base at some point when I am able